src/ast/spans.rs

```diff
@@ -40,28 +41,85 @@ pub trait Spanned {
 
 impl Spanned for Query {
     fn span(&self) -> Span {
-        self.body
-            .span()
-            .union_opt(&self.with.as_ref().map(|i| i.span()))
+        let Query {
+            with,
+            body,
+            order_by,
+            limit,
+            limit_by,
+            offset,
+            fetch,
+            locks: _,         // todo
+            for_clause: _,    // todo, mysql specific
```

Comment:
@Nyrox : `for_clause` is specific to microsoft sql server (mssql), not mysql :)

Reply:
oops 😅 , fixed